allennlp/tests/data/vocabulary_test.py

```diff
@@ -20,6 +19,8 @@
 
 
 class TestVocabulary(AllenNlpTestCase):
+    # pylint: disable=no-self-use, invalid-name, too-many-public-methods, protected-access
```

Comment:
undo?

Reply:
too-many-public-methods was added for number of public methods in test file. I mean there are >20 test methods so. It has nothing to do with public methods in `vocabulary.py`

Reply:
I just moved it from the top of the file to inside the class.

There are good principled reasons for making the disables more tightly scoped, but I did it because (for some reason) my local pylint keeps ignoring disables in the first line of a file